authors/apps/articles/urls.py

```diff
@@ -2,12 +2,13 @@
 from django.urls import path
 
 from .views import (
-    ListCreateArticleAPIView, RetrieveUpdateArticleAPIView
+    ListCreateArticleAPIView, RetrieveUpdateArticleAPIView, 
+    LikesDislikesView
 )
 
 urlpatterns = [
     path('articles/', ListCreateArticleAPIView.as_view()),
     path('articles/<slug:slug>/', RetrieveUpdateArticleAPIView.as_view()),
     path('articles/', include(('authors.apps.comments.urls', 'comments'), namespace='comments')),
+    path('articles/like-dislike/<slug:slug>/', LikesDislikesView.as_view(), name = 'likes'),
 ]
```

Comment:
This looks a good URL path **articles/<slug:slug>/like-dislike/** than this **articles/like-dislike/<slug:slug>/**

Reply:
done fixing it

Reply:
I have added the dislike route